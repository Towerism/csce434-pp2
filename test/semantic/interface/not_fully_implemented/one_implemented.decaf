class Asdf implements Qwer {
          int a(bool b) {
          }
}

interface Qwer {
          int a(bool b);
          bool b(int c);
          string c(bool d);
}
