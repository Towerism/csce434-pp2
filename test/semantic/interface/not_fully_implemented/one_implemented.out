
*** Error line 1.
class Asdf implements Qwer {
                      ^^^^
*** Class 'Asdf' does not implement entire interface 'Qwer'

